### ISSUE TITLE
Latest release (v0.6.9) doesn't match with documentation regarding self-signed certificates

### ISSUE THREAD

**Issue by mainakae** [flatiron/cradle#285]
First and foremost: thanks for this amazing library!, it saved me tones of work!.

Now a quick (and hopefully easy to fix ;) ) issue: I've been banging my head for some two days now, trying to figure out why cradle won't make any difference wether I configure the self-signed certificate or not, using the "ca" param stated in the documentation.

Tried to debug from the sources in node_modules and found that there was no code using the parameter, although there certainly was in the github repo.

Long story short: the latest release available, doesen't contain that code, thus doesn't match the documentation as seen in github, leading to the error at hand.

Fixed easy and dirty removing the dependency and reinstalling with "npm install --save git+https://github.com/flatiron/cradle.git#632a9fededb774ba5ecb94d3a268280f1bb62a88" (latest commit at the moment). All working now! (though it would be better to avoid that hack ;) )

Thanks again and happy coding!

**Comment by panuhorsmalahti** [comment]
Yeah, we haven't had a release in a while. 0.7.0 should be released soon.

**Comment by mainakae** [comment]
Now, THAT was FAST!

Thanks!!

**Comment by KingScooty** [comment]
The package json is referring to v0.7.1, but the latest github release is v.0.6.9?
